README.md

```diff
@@ -1,25 +1,56 @@
-# must-gather for Red Hat OpenShift Data Science
+# must-gather for Red Hat OpenShift AI
 
 The must-gather script allows a cluster admin to collect information about various key resources and namespaces
 for Red Hat OpenShift Data Science.
 
 ## Data Collected
 
-The must-gather script currently collects data from following namespaces
+The must-gather script currently collects data from following default namespaces
 
 - redhat-ods-operator
 - rhods-notebooks
 - redhat-ods-applications
 - redhat-ods-monitoring
 
-The script collects data from all the namespaces that has -
-- `DataSciencePipelinesApplication` instances
+and datasciencecluster and dscinitialization instances from cluster
+
+This script also collects data from all the namespaces that has
+
+- `DataSciencePipelinesApplication` instances for Data Science Pipeline component
+- `appwrappers` `quotasubtrees` `gschedulingspecs` for Codeflare component
+- `rayclusters` `rayjobs` `rayservices` for KubeRay component
+- `clusterqueues` `localqueues` `multikueueclusters` `multikueueconfigs` `provisioningrequestconfigs` `resourceflavors` `workloads` `workloadpriorityclasses` for Kueue component
+- `notebooks` for Workbench component
+- `inferenceservice` `inferencegraphs` for Kserve and ModelMesh component
 
 ## Usage
 
+To collect all
+
 ```
 oc adm must-gather --image=quay.io/modh/must-gather:stable
 ```
+
+To collect for only one component  use env variable COMPONENT.
+for example to 'kserve':
```

Comment:
Can you list somewhere in this doc the list of the components?

Reply:
added a new table for supported value and component name